plugins/verifier/sample/sample.go

```diff
@@ -37,7 +39,18 @@ type PluginInputConfig struct {
 }
 
 func main() {
+	// create a plugin logger
+	pluginlogger := logger.NewLogger()
+
+	// output to stderr
+	pluginlogger.Info("initialized plugin")
+	pluginlogger.Debugf("plugin %s %s", "sample", "1.0.0")
+
 	skel.PluginMain("sample", "1.0.0", VerifyReference, []string{"1.0.0"})
+
+	// set output to stdout
+	pluginlogger.SetOutput(os.Stdout)
+	pluginlogger.Warn("example warning message...")
```

Comment:
should we have an example on how to log an error?

Reply:
Actually looking at the latest patterns, we have been returned the error as a verifier report. Example here: https://github.com/deislabs/ratify/blob/main/plugins/verifier/vulnerabilityreport/vulnerability_report.go#L95C2-L102C3

Reply:
Interesting, looks like all the terminating errors are caught by this logic calls the [pluginErr](https://github.com/deislabs/ratify/blob/3757608fb600ab30e8c02be673b52f698a0e106b/pkg/common/plugin/exec.go#L100) function in `exec.go`  

Is the suggestion to update that to return a verifier report with the error like in the sample you provided?

Reply:
Yes ,its encourage to return the error as a part of verifier report. But incase that are missed cases, the exec should print out the error that were not a part of the verifier report.